grails-app/assets/javascripts/streama/translations/DE_de.js

```diff
@@ -3,7 +3,137 @@
+		LOGIN: {
+			TITLE: 'Bitte einloggen',
+			USERNAME: 'Benutzername',
+			PASSWORD: 'Passwort',
+			FIRST_TIME_HINT: 'Erstes mal hier? Versuche \'admin\' für Benutzername und Passwort.',
+			SUBMIT: 'Login'
+		},
+		DASHBOARD: {
+			TITLE: 'Tafelansicht',
+			NEW_RELEASES: 'Neuerscheinungen',
+			CONTINUE_WATCHING: 'Weiter ansehen',
+			DISCOVER_SHOWS: 'Entdecke Serien',
+			DISCOVER_MOVIES: 'Entdecke Filme',
+			DISCOVER_OTHER_VIDEOS: 'Entdecke Videos',
+			SORT: 'Sortiere:',
+			SEARCH_BY_NAME: 'Nach Namen suchen...',
+			FILTER_BY_TAG: 'Mit Tags filtern...',
+			BROWSE_GENRES: 'Durchsuchen',
+			LOOKING_AT_GENRE: 'Ausgewähltes Genre:',
+			MARK_COMPLETED: 'Markiere als angesehen',
+			NO_TVSHOWS_FOUND: 'Keine Serien verfügbar',
+			NO_MOVIES_FOUND: 'Keine Filme verfügbar'
+		},
+		VIDEO: {
+			RELEASED: 'Veröffentlicht',
+			IMDB: 'IMDB',
+			RATING: 'Bewertung',
+			VOTES: 'Stimmen',
+			OVERVIEW: 'Überblick',
+			GENRE: 'Genre',
+			TRAILER: 'Vorschau',
+			SEASON: 'Staffel'
+		},
 
+		MESSAGES: {
+			SHARE_SOCKET: 'Mit dem erstellen einer neuen Sesion wirst du zurück zu diesem Player geleitet, aber du bekommst eine eindeutige ID. Gib die ID an deine Freunde weiter um mit deinen Freunden synchron diesen Film zu schauen!',
+			FILE_MISSING: 'Es gibt ein Problem mit dem Videoinhalt. Es scheint als sei die Videodatei des Datenbankeintrages entfernt worden',
+			CODEC_PROBLEM: 'Es gibt ein Problem beim hinzufügen des Videos zum Player. Dies ist meist ein Codec- Problem. Versuche deine Videos in einen zu HTML5 kompatieblen Codec zu konvertieren. Lösche die aktuelle Datei und füge die recodierte hinzu. Wenn der codec korrekt ist, überprüfe die Error-Logs und die Servereinstellungen.',
+			WRONG_BASEPATH: 'Dein Video wird mit einem falsche Basispfad eingefügt. Du benutzt diese Seite über "{{basePath}}". Vergewissere dich, ob der korrekte Basispfad in den Einstellungen hinterlegt ist, und du diesen benutzt um das Video an zu sehen.'
+		},
+		MANAGE_CONTENT: 'Inhalte verwalten',
+		ADMIN: 'Verwaltung',
+		HELP: 'Hilfe',
+		HELP_FAQ: 'HILFE / FAQ',
+		PROFILE_SETTINGS: 'Profil Einstellungen',
+		LOGOUT: 'Ausloggen',
+		CHANGE_PASSWORD: 'Passwort ändern',
+		LANGUAGE_en: 'English',
+		LANGUAGE_de: 'Deutsch',
+    LANGUAGE_fr: 'French',
+		PROFIlE: {
+			USERNAME: 'Benutzername',
+			FULL_NAME: 'Vollständiger Name',
+			LANGUAGE: 'Sprache',
+			PAUSE_ON_CLICK: 'Pausiere Video bei Klick',
+			FAVORITE_GENRES: 'Lieblings-Genres',
+			SAVE: 'Profil speichern',
+			OLD_PASS: 'Altes Passwort',
+			NEW_PASS: 'Neues Passwort',
+			NEW_PASS_PLACEHOLDER: 'Neues Passwort  (min. 6 Zeichen)',
+			REPEAT_PASS: 'Passwort wiederholen',
+			SAVE_PASS: 'Neues Passwort setzen'
+		},
+
+		SORT_OPTIONS: {
+			AZ: 'A-Z',
+			ZA: 'Z-A',
+			NEWEST_ADDED: 'Zuletzt hinzugefügt',
+			OLDEST_ADDED: 'Zuerst hinzugefügt',
+			NEWEST_RELEASED: 'Neuste Veröffentlichung',
+			OLDEST_RELEASED: 'Älteste Veröffentlichung',
+			NEWEST_AIRED: 'Zuletzt angeschaut',
```

Comment:
@dularion Could you tell me if the variable NEWEST_AIRED is used to display "Last watched" or "Last aired"?